### PR TITLE
Bug/sui ssr update npm lock json

### PR DESCRIPTION
**Pull request by carlosvillu**

### CODE REVIEW

File: packages/sui-ssr/README.md [hunk 1]
```diff
@@ -72,6 +72,73 @@ $ npx sui-ssr release --email bot@email.com --name BotName
...
+        - set -e
+        - 'if [ -z $(git tag --points-at $TRAVIS_COMMIT) ]; then travis_terminate; fi'
+      name: "Deploy dev"
+      script:
+        - echo "Esto construye $NODE_ENV con la versión $TRAVIS_TAG ($TRAVIS_COMMIT_MESSAGE)"
+        - npm install surge
+        - npm install --only pro
+        - npm install --only dev
+        - npm run ssr:deploy:development
+    - #stage: deploy pro
```
Comment (jcger) [inline packages/sui-ssr/README.md]: is this really a comment?

File: packages/sui-ssr/README.md [hunk 2]
```diff
@@ -72,6 +72,73 @@ $ npx sui-ssr release --email bot@email.com --name BotName
 
 To use this command you have to define a `GH_TOKEN` env var in your CI server. This token must be associate to the user and email passing by flags to the command
 
+Example of a `.travis.yml`:
+
+- Avoid jobs for branch with the format `vX.Y.Z`
+- Avoid execute the release job if the commit has a tag associate
```
Comment (jcger) [inline packages/sui-ssr/README.md]: typo associated, next line too

File: packages/sui-ssr/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/ssr",
-  "version": "5.10.0",
+  "version": "5.10.0-beta.4",
```
Comment (jcger) [inline packages/sui-ssr/package.json]: did you forget this?

File: packages/sui-ssr/README.md [hunk 4]
```diff
@@ -72,6 +72,73 @@ $ npx sui-ssr release --email bot@email.com --name BotName
...
+- Avoid execute the deploy jobs if the commit has not a tag associate
+- Avoid a global install
+
+```
```
Comment (jcger) [inline packages/sui-ssr/README.md]: you could `yml` here for syntax highlighting

File: packages/sui-ssr/README.md (hunk 1 above)
Comment (carlosvillu) [reply packages/sui-ssr/README.md]: yep, It is to create a stage without a name.

File: packages/sui-ssr/package.json (hunk 3 above)
Comment (carlosvillu) [reply packages/sui-ssr/package.json]: This will be overridden with the release

**Pull request merged**